app/assets/stylesheets/style.scss

```diff
@@ -867,4 +873,21 @@ label:hover {
   }
 }
 
+.notice_wrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2000;
+  width: 100%;
+  background: rgba(137, 59, 100, 0.7);
+  .notice, .alert {
```

Comment:
Rule declaration should be preceded by an empty line<br>Each selector in a comma sequence should be on its own line